flocker/acceptance/scripts/lsblkhttp.py

```diff
@@ -0,0 +1,36 @@
+"""
+HTTP server that returns the size of the mounted volume.
+"""
+
+from subprocess import check_output
+
+from BaseHTTPServer import BaseHTTPRequestHandler, HTTPServer
+
+
+class Handler(BaseHTTPRequestHandler):
+    def do_GET(s):
+        s.send_response(200)
+        s.end_headers()
+        if "is_http" in s.path:
```

Comment:
I guess I wasn't clear with what I meant. Looking at the test, this is query to ensure that the container is running (as far as I can tell), so is seems like the path should reflect that.

Looking at the test a bit closer, though; if you reversed the order of the assertions, (so the the size reported by the container is checked first), then the that assertion could perhaps take the place of the one involving this path.